src/views/include/check-your-requests/collaborator.html

```diff
@@ -1,12 +1,12 @@
 {% set previousPageQueryParam = "?" + PARAMS.previousPage + "=" + ROUTES.CHECK_YOUR_REQUESTS_URL %}
 
-{% for collaborator in add_collaborator %}
+{% for collab in collaborator %}
 
-  {% set firstNameHref = "/add-collaborator/" + collaborator.id + previousPageQueryParam + "#first_name" %}
-  {% set lastNameHref = "/add-collaborator/" + collaborator.id + previousPageQueryParam + "#last_name" %}
-  {% set githubHandleHref = "/add-collaborator/" + collaborator.id + previousPageQueryParam + "#github_handle" %}
-  {% set emailAddressHref = "/add-collaborator/" + collaborator.id + previousPageQueryParam + "#email_address" %}
-  {% set repoHref = "/add-collaborator/" + collaborator.id + previousPageQueryParam + "#repo_name" %}
+  {% set firstNameHref = "/github/update/collaborator/" + collaborator.id + previousPageQueryParam + "#first_name" %}
```

Comment:
`id` is undefined, variable name has to be change to `collab`. It should be `collab.id` some for the other reference